### PR TITLE
implementing orders endpoint

### PR DESCRIPTION
**Pull request by pedrohma**

### CODE REVIEW

File: src/walmart-msw.js [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
 const axios = require('axios');
 let httpClient = null;
 
-module.exports = function(clientId, secretKey, host, version, name){
-
-    //Initialize httpClient custom Axios Instance for WMS
+module.exports = function (clientId, secretKey, version, name) { // Initialize httpClient custom Axios Instance for WMS
     httpClient = axios.create({
-        baseURL: host,
+        baseURL: 'https://marketplace.walmartapis.com/',
```
Comment (drbarzaga) [inline src/walmart-msw.js:7]: @pedrohma  Because you use the marketplace directly here?

Comment (pedrohma) [reply src/walmart-msw.js:7]: Because the url is a constant, but the version is not. Agree?

Comment (drbarzaga) [reply src/walmart-msw.js:7]: Ok you are ok

Comment (drbarzaga) [reply src/walmart-msw.js:7]: Yes you are ok, but the package can be used with the sandbox mode, and in this case, the URL is for production hardcode.

Comment (pedrohma) [reply src/walmart-msw.js]: Agreeded @drbarzaga. What do you think about [this](https://github.com/drbarzaga/walmart-mws/pull/1/commits/97a5f218faff0e85703ab972c98df37513026d10)?

Comment (drbarzaga) [reply src/walmart-msw.js]: Cool a good solution!

**Comment by drbarzaga** [comment]
@pedrohma You can update the doc of the package for example add the Orders to the content table and add one example of using the endpoint similar to this
https://www.npmjs.com/package/walmart-mws

**Comment by drbarzaga** [comment]
@pedrohma Any suggestions about mark the orders as shipped or that?

**Comment by pedrohma** [comment]
> @pedrohma Any suggestions about mark the orders as shipped or that?

Yes, that's what I will be working on. I'll open another PR later.

**Comment by drbarzaga** [comment]
@pedrohma Ok cool can you send me any feedback about the package?

**Comment by pedrohma** [comment]
> @pedrohma Ok cool can you send me any feedback about the package?

You've done an amazing job. Let's finish this API in Node. Will be very helpful for the community.

**Comment by drbarzaga** [comment]
@pedrohma Good thank you very much.

**Pull request merged**